### DISCUSSION TITLE
Unable to push commit, error giving me "The remote disconnected. Check your Internet connection and try again."

### DISCUSSION THREAD

**Discussion by GabrielaGregoris** [community/community#146868] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Unable to push commit, error giving me "The remote disconnected. Check your Internet connection and try again."  and i'm able to get my website updates to update on github link. When viewed on my computer everything is updated and working fin but not on the git hub link

**Comment by MindfulLearner** [comment]
try to `git remote -v` to see if is connected 
if not `git remote set-url origin <link>`
